tests/unit/lib/NavLink-test.js

```diff
@@ -397,19 +397,4 @@ describe('NavLink', function () {
             }, 50);
         });
     });
-
-    describe('componentWillUnmount', function () {
-        it('should update active state', function () {
-            var div = document.createElement('div');
```

Comment:
when we remove handleRoute (also connectToStore) I think we no longer listen to RouteStore change and remove listener